EU4toV2/Source/V2World/Country/Country.cpp

```diff
@@ -127,7 +127,7 @@ void V2::Country::initFromEU4Country(
 	// Localization
 	localisation.setTag(tag);
 	localisation.readFromCountry(*srcCountry);
-
+	
```

Comment:
Delete extra tab

Reply:
So, press backspace to delete, press arrow down to move to next row, and whaddayaknow, tab is back, courtesy of some VS code layout feature.

deleted.